libr/anal/sign.c

```diff
@@ -292,6 +304,35 @@ static bool addItem(RAnal *a, RSignItem *it) {
 	return retval;
 }
 
+static bool addHash(RAnal *a, const char *name, ut64 size, const ut8 *bytes) {
+	bool retval = false;
+	if (!bytes) {
+		return false;
+	}
+	
+	RSignItem *it = r_sign_item_new ();
+	if (!it) {
+		goto beach;
+	}
+	
+	it->name = r_str_new (name);
+	if (!it->name) {
+		goto beach;
+	}
+
+	char *c = r_hash_to_string (NULL, "md5", bytes, size);
+	if (c) {
```

Comment:
same here please
```
if (!c) {
   goto beach;
}

it->hash = c;
```

It helps to maintain the "main" code on the same indentation level.